.github/workflows/ui.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           cache: pnpm
           cache-dependency-path: ui/pnpm-lock.yaml
       - run: pip install -r ../requirements.txt
+      - run: pip install gradio
```

Comment:
@abidlabs added this in order to run the `create_demo_configs.py` file, which creates the configs for the integration tests. Confused how the tests were passing without this.